index.html

```diff
@@ -12,15 +11,22 @@
     <title>My Portfolio</title>
 </head>
 <body>
-        <header class="main-header"> 
-            <a class="logo" href="#"><img src="images/My Logo (1).png" alt="home page logo"></a>
-            <nav id="nav-container">
-                    <a class="desk" href="#" >Project</a>
-                    <a class="desk" href="#" >About</a> 
-                    <a class="desk" href="#" >Contact</a>
-            <a class="menu" href="#" ><img src="icon/menu.png"alt="menu"></a>    
-        </nav>
-    </header>
+        <header>
+           <a class="logo" href="#"><img src="images/My Logo (1).png" alt="home page logo"></a>
+            <div class="menu-toggle">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+            <nav>
+            <button class="close-button">&times;</button>
+              <ul class="links">
+                <li><a href="#portfolios">Portfolio</a></li>
+                <li><a href="#about-section">About</a></li>
+                <li><a href="#contacts-form">Contact</a></li>
+              </ul>
+            </nav>
+          </header>
```

Comment:
- [ ] **[OPTIONAL]** Good job so far 👏 👏 However, it would be better to make the following changes to replicate the Figma design as much as possible:
    - You may try to use the same ```font-size, font-weight and font``` to your menu and you may also try to align the menu as given by the Figma design for the best user experience 👍
    - Kindly, It would be better also to blur your background to increase the beauty and focus on your menu list 👍 

   _You may follow this [link](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/blur) for  more_ 

![image](https://user-images.githubusercontent.com/102757126/233156437-f2c268b1-14ff-415d-aa0f-49d325fadc64.png)